.github/CODEOWNERS

```
@@ -1,14 +1,26 @@
 *                                                  @plexoos           @nigmatkulov
-/OnlTools                                          @kehw              @plexoos
+/OnlTools                                          @kehw              @plexoos          @jml985
 /StDb                                              @dmarkh            @plexoos
-/StRoot/RTS                                        @kehw              @akioogawa
+/StDb/idl                                          @akioogawa         @fgeurts          @iraklic        @yfisyak
```

```suggestion
/StDb/idl                                          @akioogawa @dmarkh         @fgeurts          @iraklic @plexoos        @yfisyak
```
To prevent removing people from maintainer list

---

Actually,   a question that both @dmarkh and @plexoos are granted the permission to /StDb, does this also include all subdirectories or they have to be explicitly included when specifying the code owners for subdirectories?

---

Is it proposed to add the following people to /StDb/idl maintainers list?
 @akioogawa @dmarkh @fgeurts @iraklic @plexoos @yfisyak

Note that IDL file changes must be accompaned by database changes (by StDbLib design), therefore I need to be a reviewer for most (or all) StDb/idl requests, I guess. It is okay to commit an IDL file for the development purposes - in this case there is no corresponding database table (usually there is a test ROOT file), but for production IDLs I just have to be aware of those commits.

---

> Actually, a question that both @dmarkh and @plexoos are granted the permission to /StDb, does this also include all subdirectories or they have to be explicitly included when specifying the code owners for subdirectories?

No, the names need to be specified for subdirectories as well.

---

I appreciate all proposals involving my name but my final decision to take or not ownership of certain modules will heavily depend on how the situation with #36 evolves